src/test/java/uk/gov/hmcts/darts/cases/service/impl/AdminCaseServiceImplTest.java

```
@@ -0,0 +1,160 @@
+package uk.gov.hmcts.darts.cases.service.impl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import uk.gov.hmcts.darts.cases.controller.CaseController;
+import uk.gov.hmcts.darts.cases.model.AdminCaseAudioResponseItem;
+import uk.gov.hmcts.darts.cases.service.CaseService;
+import uk.gov.hmcts.darts.common.entity.CourtCaseEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.common.entity.MediaLinkedCaseEntity;
+import uk.gov.hmcts.darts.common.repository.MediaRepository;
+import uk.gov.hmcts.darts.test.common.data.PersistableFactory;
+import uk.gov.hmcts.darts.util.pagination.PaginatedList;
+import uk.gov.hmcts.darts.util.pagination.PaginationDto;
+
+import java.util.List;
+
+import static org.apache.commons.collections4.ListUtils.emptyIfNull;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class AdminCaseServiceImplTest {
+
+    @Mock
+    private CaseService caseService;
+    @Mock
+    private MediaRepository mediaRepository;
+
+    private AdminCaseServiceImpl adminCaseService;
+
+    @BeforeEach
+    void setUp() {
+        adminCaseService = new AdminCaseServiceImpl(caseService, mediaRepository);
+    }
```

Can use @InjectMocks instead of beforeEach

---

@InjectMocks hides errors, it is better to create the class by passing the mocks.

---

Inject mocks is standard across the app, but if it hides errors we probs need a chat around it one for the Best Practice chat maybe

---

Its already in the best practice: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1694706043#BestPractices(Backend)-FavourExplicitClassInstantiationOver@InjectMocks

---

Please stick with best practice guidance

---

I did not know that easy :)